examples/failure-cases/syntax-error/settings.gradle.kts

```diff
@@ -3,8 +3,6 @@ rootProject.name = "syntax-error"
 pluginManagement {
     repositories {
         mavenLocal()
-        // Uncomment these to use the published version of the plugin from your preferred source.
```

Comment:
Comment is not needed b/c these are failure integ tests and customers should not be copying them